packages/ui/src/components/Label/Label.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactText } from 'react';
+import { FormLabel, forwardRef, HTMLChakraProps } from '@chakra-ui/react';
+
+export type LabelSizeProps = 'sm' | 'md' | 'lg';
```

Comment:
There is no need to export this type as we don't use it outside of this component